### PR TITLE
🪟 🎨 Minor design updates

### PR DESCRIPTION
**Pull request by teallarson**
## What
Action items from weekly design pairing meeting.

- Adds hover state to Connections, Sources, and Destinations tables:
Screen Shot 2022-09-08 at 10 09 11 AM

- Sign out button is now evenly left-aligned
Screen Shot 2022-09-08 at 10 17 43 AM

## How
- added a `:hover` selector to the table rows
- removed a check that was no longer needed to fix padding problem

### CODE REVIEW

File: airbyte-webapp/src/components/ContentCard/ContentCard.tsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const Title = styled(H5)<{ light?: boolean; roundedBottom?: boolean }>`
 const ContentCard: React.FC<IProps> = (props) => (
   <Card className={props.className} onClick={props.onClick} full={props.full}>
     {props.title ? (
-      <Title light={props.light || !props.children} roundedBottom={!!props.children}>
```
Comment (tealjulia) [inline airbyte-webapp/src/components/ContentCard/ContentCard.tsx:26]: The only implementation without children was the Sign Out card, which Nico noticed looks "off" with the different padding for `light`

File: airbyte-webapp/src/components/Table/Table.tsx [hunk 2]
```diff
@@ -33,6 +33,7 @@ const TableView = styled(Card).attrs({ as: "table" })<{ light?: boolean }>`
   width: 100%;
   max-width: 100%;
   border-radius: 10px;
+  overflow: hidden;
```
Comment (edmundito) [inline airbyte-webapp/src/components/Table/Table.tsx:36]: What happens when this overflow is hidden and the screen size is small?

File: airbyte-webapp/src/components/Table/Table.tsx [hunk 3]
```diff
@@ -43,6 +44,9 @@ const Tr = styled.tr<{
 }>`
   background: ${({ theme, erroredRows }) => (erroredRows ? theme.dangerTransparentColor : theme.whiteColor)};
   cursor: ${({ hasClick }) => (hasClick ? "pointer" : "auto")};
+  &:hover {
+    background-color: ${({ theme }) => `${theme.greyColor0}`};
```
Comment (edmundito) [inline airbyte-webapp/src/components/Table/Table.tsx:48]: The legacy colors are there to support the existing code, but the new colors are also available:

```suggestion
    background-color: ${({ theme }) => `${theme.grey50}`};
```

**Review by edmundito — COMMENTED** [review]
Had a question and a suggestion

File: airbyte-webapp/src/components/Table/Table.tsx (hunk 2 above)
Comment (tealjulia) [reply airbyte-webapp/src/components/Table/Table.tsx:36]: Oops!  This should have been removed, thanks.  Was part of trying to get the sticky headers to work.  I've split that into its own issue #16435 .

**Review by edmundito — APPROVED** [review]

**Pull request merged**